src/routes/routes.js

```diff
@@ -9,16 +9,35 @@ import {
 } from '../controllers/commentController';
 import { authenticateUserToken } from '../middlewares/authentication';
 import { upvoteInvite } from '../controllers/upvoteController';
+import { validateAdmin} from '../middlewares/validateAdmin'
```

Comment:
You see this validateAdmin should be imported in the `middlewares.js` file. Then you will now reference it here through the `middlewares.js` file. You can check other middleware were implemented.